src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/NodeMonitoringConnectionPlugin.java

```diff
@@ -182,13 +180,28 @@ public Object execute(String methodName, Callable executeSqlFunc) throws Excepti
         executor.shutdownNow();
       }
       this.log.logTrace(String.format(
-          "[NodeMonitoringConnectionPlugin.execute]: method=%s, monitoring is deactivated",
-          methodName));
+          "[NodeMonitoringConnectionPlugin.execute]: method=%s.%s, monitoring is deactivated",
+              methodInvokeOn.getName(), methodName));
     }
 
     return result;
   }
 
+  protected boolean isNeedMonitoring(Class methodInvokeOn, String methodName) {
```

Comment:
`Class methodInvokeOn` unused in code block

Reply:
It is unused. Just wanted to keep pieces of "sample" code to show how to deal with methodInvokeOn parameter.